src/test/regress/sql/multi_extension.sql

```diff
@@ -165,6 +165,14 @@ WHERE pgd.refclassid = 'pg_extension'::regclass AND
 	  pge.extname    = 'citus' AND
 	  pgio.schema    NOT IN ('pg_catalog', 'citus', 'test');
 
+-- if the output of following query changes, we might need to change
```

Comment:
I think maintaining an extra output file for `multi_extension.sql` could be very painful because we need to update it very frequently, whenever we add/remove/change UDFs. 

What if we add a new test file, `multi_metadata_attributes.sql` just before `multi_extension.sql` and only includes the followin:

```SQL
CREATE EXTENSION citus;

 -- the query you've -- 

DROP EXTENSION citus;
```

so that no one / no test actually is affected by this on regular basis?